.storybook/preview.js

```diff
@@ -15,10 +15,12 @@ import { createVueless } from "../src/index.js";
 import { createRouter, createWebHistory } from "vue-router";
 
 /* Setup storybook */
-setup((app) => {
+setup((app, context) => {
   const vueless = createVueless();
   const router = createRouter({ history: createWebHistory(), routes: [] });
 
+  app.config.idPrefix = context.name;
```

Comment:
For what reason this line is needed?

Reply:
Each story is vue app instance. We have to define id prefix for useId when we have more than one app in the page.